spec/jobs/regular/jira_post_liked_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Jobs::JiraPostLiked do
+  fab!(:category)
+  fab!(:topic) { Fabricate(:topic, category: category) }
+  fab!(:post) { Fabricate(:post, topic: topic, post_number: 1, like_count: 3) }
+  subject(:execute) { job.execute(post_id: post.id) }
+
+  let(:job) { described_class.new }
+
+  before do
+    SiteSetting.discourse_jira_enabled = true
+    SiteSetting.discourse_jira_url = "https://test.atlassian.net"
+    category.custom_fields["jira_num_likes_auto_create_issue"] = 3
+    category.custom_fields["jira_project_id"] = 1
+    category.custom_fields["jira_issue_type_id"] = 1
+    category.save_custom_fields
+  end
+
+  it "does nothing if the post is not the first post" do
+    post.update!(post_number: 2)
+    DiscourseJira::IssueCreator.expects(:create).never
+    execute()
```

Comment:
```suggestion
    execute
```

No need for `()` on all of these :P